Geometry_Engine/Compute/DistributionVector.cs

```diff
@@ -0,0 +1,52 @@
+/*
+ * This file is part of the Buildings and Habitats object Model (BHoM)
+ * Copyright (c) 2015 - 2022, the respective contributors. All rights reserved.
+ *
+ * Each contributor holds copyright over their respective contributions.
+ * The project versioning (Git) records all such contribution source information.
+ *                                           
+ *                                                                              
+ * The BHoM is free software: you can redistribute it and/or modify         
+ * it under the terms of the GNU Lesser General Public License as published by  
+ * the Free Software Foundation, either version 3.0 of the License, or          
+ * (at your option) any later version.                                          
+ *                                                                              
+ * The BHoM is distributed in the hope that it will be useful,              
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of               
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the                 
+ * GNU Lesser General Public License for more details.                          
+ *                                                                            
+ * You should have received a copy of the GNU Lesser General Public License     
+ * along with this code. If not, see <https://www.gnu.org/licenses/lgpl-3.0.html>.      
+ */
+
+using BH.Engine.Base;
+using BH.oM.Geometry;
+using BH.oM.Base.Attributes;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.ComponentModel;
+
+namespace BH.Engine.Geometry
+{
+    public static partial class Compute
+    {
+        /***************************************************/
+        /****          Public Methods                   ****/
+        /***************************************************/
+
+        [Description("Returns the distribution vector for element placement, dependent upon the number of elements and the start and end points.")]
+        [Input("elementCount", "Number of elements to place between the two points.")]
+        [Input("startPoint", "The start point of the distribution vector to create the Symbols.")]
+        [Input("endPoint", "The end point of the distribution vector to create the Symbols.")]
+        [Output("distributionVector", "The distribution vector with the correct length to create the Symbols.")]
+        public static Vector DistributionVector(Vector vector, int elementsCount = 1)
```

Comment:
In general, I would phrase the descriptions around placing points, not elements/items - since we are in Geometry_Engine, let's stick to geometry 😉 

```suggestion
        public static Vector DistributionVector(Vector vector, int pointCount = 1)
```